tests/test_quantity.py

```diff
@@ -0,0 +1,398 @@
+import pytest
+
+import ansys.fluent.core.quantity as q
+
+
+def test_viscosity():
+    v = q.Quantity(1, "P")  # poise
+    conversion_output = v["kg m^-1 s^-1"]
+    assert conversion_output._real_value == 0.1
+
+
+def test_volume():
+    v = q.Quantity(1, "gal")
+    conversion_output = v["m^3"]
+    assert conversion_output._real_value == pytest.approx(0.00378541)
+
+
+def test_youngs_modulus():
+    ym = q.Quantity(1, "lbf ft^-2")
+    conversion_output = ym["N m^-2"]
+    assert conversion_output._real_value == pytest.approx(47.89, 0.1)
+
+
+def test_temperature():
+    tempC = q.Quantity(1, "degC")  # celsius
+    conversion_output_1 = tempC["degK"]
+    assert conversion_output_1._real_value == 274.15
```

Comment:
Why do you have to access a private variable to test this?  Would you not have an API `conversion_output_1.value()` or a public property `conversion_output_1.value`.  I don't think you should name it `real_value` since it might imply later that there is the possibility of a `complex_value`.

Reply:
_real_value changed to value.